s2i/bin/run

```diff
@@ -23,6 +23,8 @@ echo "---> \$WEB_PATH is set to: ${WEB_PATH}"
 if [ "${WEB_PATH}" == "/" ]; then
   echo "---> Unsetting \$WEB_PATH to avoid an apache alias being created."
   unset WEB_PATH
+else
+  sed -i "s@# RewriteBase /\$@RewriteBase ${WEB_PATH}@g" /code/web/.htaccess
```

Comment:
I thought this wasn't necessary because of https://github.com/universityofadelaide/s2i-shepherd-drupal/blob/master/files/apache2.conf#L60 ?

Reply:
Oh, ok. Jesus that's annoying.